main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -18,6 +18,7 @@
 package ca.uwaterloo.flix.language.phase
 
 import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.language.ast.Name.QName
```

Comment:
Drop import, write Name.QName -- thats what we do everywhere else as a matter of style.